tests/pipelinesintegrations_test.go

```diff
@@ -114,7 +114,7 @@ func getIntegrationAndAssert(t *testing.T, id int, name, integrationType string)
 }
 
 func getUniqueIntegrationName(integrationType string) string {
-	return strings.Join([]string{integrationNamesPrefix, integrationType, timestampStr}, "_")
+	return strings.Join([]string{integrationNamesPrefix, integrationType, getRunId('_')}, "_")
```

Comment:
I suggest creating two functions for convenience (`getDefaultRunId()` and `getCustomRunId(separator rune)` for example).
Also please add a comment explaining why did is needed